openmls/src/framing/message.rs

```diff
@@ -30,6 +30,7 @@ use crate::error::LibraryError;
 /// ```
 #[derive(PartialEq, Debug, Clone, TlsSerialize, TlsSize, TlsDeserialize)]
 pub(crate) struct MlsMessage {
+    pub(crate) protocol_version: ProtocolVersion,
```

Comment:
You call it `version` in `FramedContentTbs`, and that's what it is in the spec as well. Maybe let's use only version?